<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>501</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(217, 229, 250);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="CanvasModel" name="canvasWidget" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>-90</x>
      <y>0</y>
      <width>761</width>
      <height>641</height>
     </rect>
    </property>
    <widget class="QPushButton" name="addFrameButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>70</y>
       <width>111</width>
       <height>25</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.0113636, x2:1, y2:1, stop:0 rgba(237, 237, 237, 255), stop:1 rgba(255, 255, 255, 255));
border: 0.5px solid rgb(178, 178, 178);</string>
     </property>
     <property name="text">
      <string>Add Frame</string>
     </property>
    </widget>
    <widget class="QLabel" name="canvas">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>140</y>
       <width>271</width>
       <height>231</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>ArrowCursor</cursorShape>
     </property>
     <property name="mouseTracking">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="pixelLocationLabel">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>440</y>
       <width>63</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
      </font>
     </property>
     <property name="text">
      <string>0, 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="previewLabel">
     <property name="geometry">
      <rect>
       <x>510</x>
       <y>40</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
    <widget class="QLabel" name="fpsLabel">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>50</y>
       <width>101</width>
       <height>20</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
      </font>
     </property>
     <property name="text">
      <string>Animation FPS: </string>
     </property>
    </widget>
    <widget class="QSlider" name="fpsSlider">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>80</y>
       <width>91</width>
       <height>31</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>60</number>
     </property>
     <property name="sliderPosition">
      <number>10</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QPushButton" name="playButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>40</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.0113636, x2:1, y2:1, stop:0 rgba(237, 237, 237, 255), stop:1 rgba(255, 255, 255, 255));
border: 0.5px solid rgb(178, 178, 178);</string>
     </property>
     <property name="text">
      <string>Play Animation</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QPushButton" name="changePenColorButton">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>40</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.0113636, x2:1, y2:1, stop:0 rgba(237, 237, 237, 255), stop:1 rgba(255, 255, 255, 255));
border: 0.5px solid rgb(178, 178, 178);</string>
     </property>
     <property name="text">
      <string>Change Pen Color</string>
     </property>
    </widget>
    <widget class="QPushButton" name="brushButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>40</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.0113636, x2:1, y2:1, stop:0 rgba(237, 237, 237, 255), stop:1 rgba(255, 255, 255, 255));
border: 0.5px solid rgb(178, 178, 178);</string>
     </property>
     <property name="text">
      <string>Brush</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eraserButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.0113636, x2:1, y2:1, stop:0 rgba(237, 237, 237, 255), stop:1 rgba(255, 255, 255, 255));
border: 0.5px solid rgb(178, 178, 178);</string>
     </property>
     <property name="text">
      <string>Eraser</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eyedropperButton">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>91</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.0113636, x2:1, y2:1, stop:0 rgba(237, 237, 237, 255), stop:1 rgba(255, 255, 255, 255));
border: 0.5px solid rgb(178, 178, 178);</string>
     </property>
     <property name="text">
      <string>Eyedropper</string>
     </property>
    </widget>
    <widget class="QPushButton" name="onionSkinningButton">
     <property name="geometry">
      <rect>
       <x>630</x>
       <y>100</y>
       <width>111</width>
       <height>29</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Trebuchet MS</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.0113636, x2:1, y2:1, stop:0 rgba(237, 237, 237, 255), stop:1 rgba(255, 255, 255, 255));
border: 0.5px solid rgb(178, 178, 178);</string>
     </property>
     <property name="text">
      <string>Previous Frame</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goBackwardButton">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>270</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.0113636, x2:1, y2:1, stop:0 rgba(237, 237, 237, 255), stop:1 rgba(255, 255, 255, 255));
border: 0.5px solid rgb(178, 178, 178);</string>
     </property>
     <property name="text">
      <string>&lt;</string>
     </property>
    </widget>
    <widget class="QPushButton" name="goForwardButton">
     <property name="geometry">
      <rect>
       <x>590</x>
       <y>270</y>
       <width>100</width>
       <height>32</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Arial Black</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:1, y1:0.0113636, x2:1, y2:1, stop:0 rgba(237, 237, 237, 255), stop:1 rgba(255, 255, 255, 255));
border: 0.5px solid rgb(178, 178, 178);</string>
     </property>
     <property name="text">
      <string>&gt;</string>
     </property>
    </widget>
    <widget class="QLabel" name="onionSkinningLabel">
     <property name="geometry">
      <rect>
       <x>-30</x>
       <y>320</y>
       <width>65</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Onion Skin</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="frameSizeSpinBox">
     <property name="geometry">
      <rect>
       <x>350</x>
       <y>440</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <property name="keyboardTracking">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>256</number>
     </property>
     <property name="value">
      <number>16</number>
     </property>
    </widget>
    <widget class="QLabel" name="frameSizeLabel">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>440</y>
       <width>61</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
      </font>
     </property>
     <property name="text">
      <string>Canvas Size:</string>
     </property>
    </widget>
    <widget class="QLabel" name="frameNumLabel">
     <property name="geometry">
      <rect>
       <x>450</x>
       <y>440</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Ebrima</family>
      </font>
     </property>
     <property name="text">
      <string>Frame 0</string>
     </property>
    </widget>
    <widget class="QLabel" name="lastFrameLabel">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>340</y>
       <width>71</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
    <zorder>canvas</zorder>
    <zorder>addFrameButton</zorder>
    <zorder>pixelLocationLabel</zorder>
    <zorder>previewLabel</zorder>
    <zorder>fpsLabel</zorder>
    <zorder>fpsSlider</zorder>
    <zorder>playButton</zorder>
    <zorder>changePenColorButton</zorder>
    <zorder>brushButton</zorder>
    <zorder>eraserButton</zorder>
    <zorder>eyedropperButton</zorder>
    <zorder>onionSkinningButton</zorder>
    <zorder>goBackwardButton</zorder>
    <zorder>goForwardButton</zorder>
    <zorder>frameSizeSpinBox</zorder>
    <zorder>frameSizeLabel</zorder>
    <zorder>frameNumLabel</zorder>
    <zorder>onionSkinningLabel</zorder>
    <zorder>lastFrameLabel</zorder>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>17</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QMenu" name="menuWindow">
    <property name="title">
     <string>View</string>
    </property>
    <addaction name="actionHelp"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave_As"/>
    <addaction name="actionSave"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuWindow"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave_As">
   <property name="text">
    <string>Save As</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionPallet">
   <property name="text">
    <string>Pallet</string>
   </property>
  </action>
  <action name="actionHelp">
   <property name="text">
    <string>Help</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CanvasModel</class>
   <extends>QWidget</extends>
   <header location="global">canvasmodel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
